db/change_cache.go

```diff
@@ -787,9 +786,8 @@ func (c *changeCache) _addPendingLogs(ctx context.Context) channels.Set {
 			heap.Pop(&c.pendingLogs)
 			changedChannels = changedChannels.UpdateWithSlice(c._addToCache(ctx, change))
 		} else if len(c.pendingLogs) > c.options.CachePendingSeqMaxNum || time.Since(c.pendingLogs[0].TimeReceived) >= c.options.CachePendingSeqMaxWait {
-			c.db.DbStats.Cache().NumSkippedSeqs.Add(1)
-			c.PushSkipped(ctx, c.nextSequence)
-			c.nextSequence++
+			c.PushSkipped(ctx, c.nextSequence, change.Sequence-1)
```

Comment:
This is a nice change (to push everything from nextSequence to the first pending sequence at once, instead of in multiple iterations of the loop).  It took me a minute to work out what exactly this was doing.  What do you think about the following, for readability:
- rename 'change' to 'oldestPending'
- Add a comment above line 789 like `//  Skip all sequences up to the oldest Pending`